server/controllers/entities/lib/entities.ts

```diff
@@ -109,8 +110,8 @@ interface PutInvEntityUpdateParams extends PutInvEntityCommonParams {
   create?: false
 }
 export async function putInvEntityUpdate <T extends InvEntityDoc = InvEntity> (params: PutInvEntityCreationParams | PutInvEntityUpdateParams) {
-  const { userId, currentDoc, updatedDoc, create, batchId, context } = params
-  assert_.types([ 'string', 'object', 'object' ], [ userId, currentDoc, updatedDoc ])
+  const { userAcct, currentDoc, updatedDoc, create, batchId, context } = params
+  assert_.types([ 'string', 'object', 'object' ], [ userAcct, currentDoc, updatedDoc ])
```

Comment:
Is'nt the point of typing every function to not have to do those validations in code anymore ?
```suggestion
```

Reply:
Some are still useful for type narrowing (ex:
```ts
function foo (a: unknown) {
  assertString(a)
  // We now know it's a string, so we can use string methods
  return a.match(/hello/)
}
```

but here that's not the case, it was just easier to update it than to purposfully think if it was needed ^^' Removed in f687ee071